apps/dot_ext/tests/test_views.py

```diff
@@ -91,6 +91,7 @@ def _create_test_token(self, user, application):
             'scope': application.scopes().split(" "),
             'expires_in': 86400,
             'allow': True,
+            'share_choice': '',
```

Comment:
should this be `block_personal_choice`?

Reply:
Removed this from here. Working on tests related to when the `require-scopes` switch is set next.